matter/wifi/lwip_netif.cpp

```diff
@@ -56,20 +64,16 @@ static void netif_config(struct netif *sta_if, struct netif *ap_if)
 
 #ifdef SL_WFX_CONFIG_SOFTAP
   if (ap_if != NULL) {
-    // ip_addr_t ap_ipaddr;
-    // ip_addr_t ap_netmask;
-    // ip_addr_t ap_gw;
-    // /* Initialize the SoftAP information */
-    // IP_ADDR4(&ap_ipaddr, ap_ip_addr0, ap_ip_addr1, ap_ip_addr2, ap_ip_addr3);
-    // IP_ADDR4(&ap_netmask, ap_netmask_addr0, ap_netmask_addr1, ap_netmask_addr2, ap_netmask_addr3);
-    // IP_ADDR4(&ap_gw, ap_gw_addr0, ap_gw_addr1, ap_gw_addr2, ap_gw_addr3);
-    // netif_add(apNetif, &ap_ipaddr, &ap_netmask, &ap_gw, NULL, &ap_ethernetif_init, &ethernet_input);
-  }
+    }
```

Comment:
Is it normal that this is an empty if statement?

Reply:
Added some comment 
this is TODO code for Intialization of SoftAP